src-self-hosted/codegen.zig

```diff
@@ -75,7 +75,7 @@ pub fn generateSymbol(
             switch (bin_file.base.options.target.cpu.arch) {
                 .wasm32 => unreachable, // has its own code path
                 .wasm64 => unreachable, // has its own code path
-                //.arm => return Function(.arm).generateSymbol(bin_file, src, typed_value, code, dbg_line, dbg_info, dbg_info_type_relocs),
+                .arm => return Function(.arm).generateSymbol(bin_file, src, typed_value, code, dbg_line, dbg_info, dbg_info_type_relocs),
                 //.armeb => return Function(.armeb).generateSymbol(bin_file, src, typed_value, code, dbg_line, dbg_info, dbg_info_type_relocs),
```

Comment:
I don't suppose you could do armeb at the same time? Seems like it should be pretty simple, just use the correct endianness when writing integers into the code array

Reply:
Ok, I'll do that as well.